src/main/java/com/microsoft/azure/spring/data/cosmosdb/core/convert/JsonSerializableFactory.java

```diff
@@ -0,0 +1,27 @@
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See LICENSE in the project root for
+ * license information.
+ */
+        
+package com.microsoft.azure.spring.data.cosmosdb.core.convert;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.datatype.jdk8.Jdk8Module;
+import com.fasterxml.jackson.datatype.jsr310.JavaTimeModule;
+import com.fasterxml.jackson.module.paramnames.ParameterNamesModule;
+
+public class JsonSerializableFactory {
```

Comment:
How about ObjectMapperFactory?

Reply:
How about move this field and method to Utils file, new class for this may too heavy here in my opinion.